eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -45,6 +51,11 @@ steps:
             $(_OfficialBuildIdArgs)
     displayName: Build
 
+  - ${{ if eq(parameters.isWindows, 'true') }}:
+    - pwsh: |
+          $(Build.SourcesDirectory)/src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/azure-devops-report/build.ps1 -OutputPath $(Build.ArtifactStagingDirectory)\VSIX
```

Comment:
We don't use `$(Build.ArtifactStagingDirectory)` as we use the Arcade.

We have few help variables defined:
https://github.com/dotnet/extensions/blob/c086106865b9808eba1acd78db1f91680952e065/azure-pipelines-public.yml#L48-L54

Reply:
Tagging @peterwald. I synced up with @russkie offline - we can handle this in the follow up PRs you will be creating for setting up for the VSIX build.